### PR TITLE
Merge SWG annotations in manually created `@Operation`

### PR DESCRIPTION
**Pull request by GuilhemN**
Fixes https://github.com/nelmio/NelmioApiDocBundle/issues/1302, https://github.com/nelmio/NelmioApiDocBundle/pull/1308

### CODE REVIEW

File: Tests/Functional/FunctionalTest.php [hunk 1]
```diff
@@ -347,4 +347,15 @@ public function testConfigReference()
         $operation = $this->getOperation('/api/configReference', 'get');
         $this->assertEquals('#/definitions/Test', $operation->getResponses()->get('200')->getSchema()->getRef());
     }
+
+    public function testOperationsWithOtherAnnotationsAction()
+    {
+        $getOperation = $this->getOperation('/api/multi-annotations', 'get');
+        $this->assertEquals('This is the get operation', $getOperation->getDescription());
```
Comment (phansys) [inline Tests/Functional/FunctionalTest.php]: Please, use `assertSame()` instead.

Comment (GuilhemN) [reply Tests/Functional/FunctionalTest.php]: Does it trully matter? We're using `assertEquals` basically everywhere.

**Comment by chregu** [comment]
Using this branch, I get the following error

``` 
User Notice: Unexpected @Nelmio\ApiDocBundle\Annotation\Model(), expected to be inside @SWG\Parameter, @SWG\Response in \OrganizationController->getAction() in 
``` 

the docblock looks like this

```
     * @Route("/organizations/{organization}",
     *     methods={"GET"},
     *     name="get_organization"
     *     }
     * )
     * @Operation(operationId="getOrganization")
     *
     * @SWG\Tag(name="admin")
     *
     * @SWG\ExternalDocumentation(url="https://rokka.io/documentation/references/organizations.html#read-data-of-one-organization")
     *
     * @SWG\Parameter(
     *     name="organization",
     *     in="path",
     *     type="string",
     *     description="Organization name"
     * )
     *
     * @SWG\Response(
     *     response=200,
     *     description="Information about the requested organization",
     *     @Model(type=Organization::class, groups={"api"})
     * )
     * @SWG\Response(
     *     response=404,
     *     description="If the organization was not found or the current user is not allowed to see that organization"
     * )
     *
     * @return Response

``` 
That worked fine in the MR of #1308

**Comment by GuilhemN** [comment]
Finally fixed!

**Pull request merged**